assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -1079,8 +1089,41 @@ public String buildMainPanelContext(VelocityPortlet portlet, Context context, Ru
         context.put("dateFormat", getDateFormatString());
         context.put("cheffeedbackhelper", this);
         context.put("service", assignmentService);
-
+        
+        // NAM-34 check if current user has marker role
```

Comment:
Hi Wandi
I'm sure Sakai should have a method already that does what you are trying to do here (getting all roles, users to check if current user has the marker permission). I'll have a look and see if I can find something that can help you with this...